java-frontend/src/test/files/se/FlowComputationRelSV.java

```diff
@@ -41,7 +41,9 @@ void catof3() {
   void catof3b() {
     Object a = new Object(); // flow@catof3b {{Constructor implies 'non-null'.}} flow@catof3b {{'a' is assigned non-null.}}
     Object b = null; // flow@catof3b {{'b' is assigned null.}}
-    boolean cond = a == b;  // no message here, because no constraint on 'a==b', it is not yet evaluated
+    boolean cond = a == b;  // no message here, because no constraint on 'a==b', it is not yet evaluated se SONARJAVA-1911
+    b = new Object(); // no message here, b is not tracked yet
+    a = null; // no message here, a is not tracked yet
```

Comment:
comment is confusing : this is not about not being tracked yet : symbol is not relevant for this assignement